src/constants.py

```diff
@@ -13,3 +13,4 @@
 
 # integrations
 S3 = "s3-credentials"
+SERVICE_ACCOUNT = "spark-service-account"
```

Comment:
nit: maybe SERVICE_ACCOUNT_RELATION_NAME or something similar would be more appropriate name since SERVICE_ACCOUNT on the first glance looks like it is name of a K8s service account.